hexrd/ui/image_canvas.py

```diff
@@ -168,10 +182,12 @@ def draw_powder_overlay(self, axis, data, style):
         data_style = style['data']
         ranges_style = style['ranges']
 
+        artists = []
+        self.overlay_artists[id(data)] = artists
         for pr in rings:
             x, y = self.extract_ring_coords(pr)
             artist, = axis.plot(x, y, **data_style)
```

Comment:
This an instance of `Line2D` a subclass of artist?

Reply:
Yep! It is a subclass of `Artist`.